lib/modules/manager/index.ts

```diff
@@ -60,7 +61,7 @@ export function extractPackageFile(
   manager: string,
   content: string,
   fileName: string,
-  config: ExtractConfig
+  config: Partial<WorkerExtractConfig>
```

Comment:
TS requires this after we have narrowed down the config as `extractPackageFile` handles config of both the types `ExtractConfig` and `CustomExtractConfig`.